modin/config/envvars.py

```diff
@@ -644,6 +644,8 @@ def put(cls, value: int) -> None:
         if value <= 0:
             raise ValueError(f"Min partition size should be > 0, passed value {value}")
         super().put(value)
+        MinRowPartitionSize.put(value)
+        MinColumnPartitionSize.put(value)
```

Comment:
I think we should limit the use of `MinPartitionSize` variable simultaneously with variables `MinRowPartitionSize` and `MinColumnPartitionSize` to avoid unexpected effects due to the order in which they are constructed from environment variables.